<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Florian D. Schneider" />


<title>Data</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">trait synthesis proposal #20</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="introduction.html">Introduction</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Methods
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="data.html">Data</a>
    </li>
    <li>
      <a href="analyses.html">Analyses</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Data</h1>
<h4 class="author"><em>Florian D. Schneider</em></h4>
<h4 class="date"><em>6 November 2017</em></h4>

</div>

<div id="TOC">
<ul>
<li><a href="#data-sources">Data sources</a><ul>
<li><a href="#abundance-data">Abundance Data</a><ul>
<li><a href="#issues">Issues:</a></li>
<li><a href="#datasets">Datasets:</a></li>
<li><a href="#code">Code:</a></li>
<li><a href="#notes">Notes:</a></li>
</ul></li>
<li><a href="#arthropod-trait-data">Arthropod Trait data</a><ul>
<li><a href="#issues-1">Issues:</a></li>
<li><a href="#datasets-1">Datasets:</a></li>
<li><a href="#code-1">Code:</a></li>
<li><a href="#notes-1">Notes:</a></li>
</ul></li>
<li><a href="#plant-traits">Plant traits</a><ul>
<li><a href="#issues-2">Issues:</a></li>
<li><a href="#datasets-2">Datasets:</a></li>
<li><a href="#code-2">Code:</a></li>
<li><a href="#notes-2">Notes:</a></li>
</ul></li>
<li><a href="#functional-soil-properties">Functional soil properties</a><ul>
<li><a href="#issues-3">Issues:</a></li>
<li><a href="#datasets-3">Datasets:</a></li>
<li><a href="#code-3">Code:</a></li>
<li><a href="#notes-3">Notes:</a></li>
</ul></li>
<li><a href="#below-ground-consumers">Below ground consumers</a><ul>
<li><a href="#issues-4">Issues:</a></li>
<li><a href="#datasets-4">Datasets:</a></li>
<li><a href="#code-4">Code:</a></li>
<li><a href="#notes-4">Notes:</a></li>
</ul></li>
<li><a href="#anthropogenic-land-use">Anthropogenic Land-use</a><ul>
<li><a href="#issues-5">Issues:</a></li>
<li><a href="#datasets-5">Datasets:</a></li>
<li><a href="#notes-5">Notes:</a></li>
</ul></li>
<li><a href="#environmental-parameters">Environmental parameters</a><ul>
<li><a href="#issues-6">Issues:</a></li>
<li><a href="#datasets-6">Datasets:</a></li>
<li><a href="#code-5">Code:</a></li>
<li><a href="#notes-6">Notes:</a></li>
</ul></li>
</ul></li>
</ul>
</div>

<div id="data-sources" class="section level1">
<h1>Data sources</h1>
<p>To analyse synchrony in ecosystem-level functional properties across multiple functional guilds in relation to land-use intensity and environmental factors, we analysed functional properties of 150 grassland plots. For three regions across Germany, 50 plots of varying land-use management have been assessed between 2008 and 2014 within the long-term framework of the Biodiversity Exploratories project.</p>
<p>For arthropods and plants, we combined functional trait data with species abundances into community weighted means. Below-ground functional properties have been assessed at the community level for the same plots.</p>
<div id="abundance-data" class="section level2">
<h2>Abundance Data</h2>
<p>Arthropod abundances were assessed in sweep-net sampling at two occasions each year (June and August) for each of the 150 plots. For this study, we are using data from 2008 to 2013. The abundances found on both sampling events were pooled into an annual value of standardised abundance per species.</p>
<p>The plant surveys took place twice each year on each of the 150 plots in a <span class="math inline">\(4\times4\)</span> m² area and estimated plant cover of each species.</p>
<div id="issues" class="section level3">
<h3>Issues:</h3>
<ul>
<li>Need to switch to new data on BExIS (21969)! (see notes)</li>
<li>Add orthoptera for 2014</li>
<li>plant abundance dataset has some name issues (ask Debi and Dani).</li>
</ul>
</div>
<div id="datasets" class="section level3">
<h3>Datasets:</h3>
<table style="width:64%;">
<colgroup>
<col width="16%" />
<col width="6%" />
<col width="15%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th>dataset</th>
<th>file</th>
<th>BExIS ID</th>
<th>author/owner</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Sweep net samples from grasslands since 2008, Araneae, Coleoptera, Hemiptera, Orthoptera</td>
<td></td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=21969">21969</a></td>
<td>Wolfgang Weisser, Martin Goßner, Nadja Simons et al.</td>
</tr>
<tr class="even">
<td>individual datasets for order and year</td>
<td><a href="https://www.dropbox.com/sh/mqyj37mzk2coo52/AABnv9KtWLQ-dQYsnHhGdHroa?dl=0">data/arthropods_core/Arthropods_GL_0813.csv</a></td>
<td>16871, 16892, 16894, 16895, 16897, 16908, 16926, 17006, 17007, 17008, 17009, 17010, 17011, 17012, 17013, 17014, 17015, 17246, 17247, 17266, 17267</td>
<td>Martin Goßner, Nadja Simons, Wolfgang Weisser et al.</td>
</tr>
<tr class="odd">
<td>Vegetation Records for Grassland EPs, 2008 - 2016</td>
<td></td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=19686">19686</a></td>
<td>Debi Schäfer, Markus Fischer</td>
</tr>
</tbody>
</table>
</div>
<div id="code" class="section level3">
<h3>Code:</h3>
<p>Code for reading and cleaning data:</p>
<ul>
<li>arthropod abundances: <a href="https://www.dropbox.com/sh/mqyj37mzk2coo52/AABnv9KtWLQ-dQYsnHhGdHroa?dl=0">/code/import_arthropod_abundance.r</a>, reads ‘data/arthropods_core/Arthropods_GL_0813.csv’ and subsets herbivore and consumer datasets.</li>
<li>vegetation cover: <a href="https://www.dropbox.com/sh/mqyj37mzk2coo52/AABnv9KtWLQ-dQYsnHhGdHroa?dl=0">/code/import_plant_abundance.r</a>, reads ‘data/plants_core/19686.txt’ selects and fixes species names, eliminates trees and shrubs.</li>
</ul>
<p>More detailled analysis of data.</p>
</div>
<div id="notes" class="section level3">
<h3>Notes:</h3>
<p>Originally, abundances for each sampling event had to be extracted from separate datasets (split by order and year, IDs: 16871, 16892, 16894, 16895, 16897, 16908, 16926, 17006, 17007, 17008, 17009, 17010, 17011, 17012, 17013, 17014, 17015, 17246, 17247, 17266, 17267), but Martin compiled a single dataset, including the years 2014 and 2015. I was using an xls file that he had transmitted inofficially, but year 2014 (2015) of arthropod sampling was missing.</p>
<p>There is an issue with the inconsistency of the number of replicates per year per plot. VIPs have been sampled four to six times, while most EPs have been sampled only twice until 2010. After 2011 they have been sampled five or six times. For calculating the community weighted means, I will have to sum the species numbers and devide by the number of replicates. When it comes to measures of variation, it will be heterogeneous just because of the difference in <span class="math inline">\(n\)</span>. I guess this means that I need to either form subsets with only two replicates per year taken into account even for VIPs. Extracting the correct and representative samplings from these data (i.e. just the samplings in June and August) is really tedious.</p>
<p>Martin confirmed this issue and also the use of the data selection process. For the data of Schorfheide, there was a shift in the sampling month in 2009, which has to be taken into account. He promised to send me a dataset that is readily compiled and includes abundance data for each plot, summed over the two comparable sampling events per year (June and August).</p>
<p>The new data (21969) came online mid 2017 and should resolve all these issues. But I havent checked them out, yet. Orthoptera from 2014 and 2015 are still missing, but will be added soon.</p>
</div>
</div>
<div id="arthropod-trait-data" class="section level2">
<h2>Arthropod Trait data</h2>
<p>Arthropod traits for all species found in the sweep-net samplings were collated from literature. The dataset contains body size, dispersal ability, and factorial feeding traits. We first split the dataset into trophic guilds of herbivores and predators based on these trait data.</p>
<p>Then, we use abundances to calculate Community weighted Means (CWM) for each guild in each year. CWMs are defined as the average individual trait values observed within a community. We argue that they represent an optimal trait value for an individual under the given biotic and abiotic conditions.</p>
<div id="issues-1" class="section level3">
<h3>Issues:</h3>
<ul>
<li>I used Gossner et al 2015 (open access data), where some taxa were missing; Nadja was working on a revised version that will be available via BExIS soon</li>
<li>define appropriate transformations to normalize traits, e.g. log-transform body size.</li>
<li>also calculate CW Variance and other metrics? Functional Diversity?</li>
</ul>
</div>
<div id="datasets-1" class="section level3">
<h3>Datasets:</h3>
<table style="width:64%;">
<colgroup>
<col width="16%" />
<col width="6%" />
<col width="15%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th>dataset</th>
<th>URL</th>
<th>BExIS ID</th>
<th>author/owner</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>A summary of eight traits of Coleoptera, Hemiptera, Orthoptera and Araneae, occurring in grasslands in Germany</td>
<td><a href="http://dx.doi.org/10.5061/dryad.53ds2" class="uri">http://dx.doi.org/10.5061/dryad.53ds2</a></td>
<td></td>
<td>Martin Gossner, Nadja Simons</td>
</tr>
</tbody>
</table>
</div>
<div id="code-1" class="section level3">
<h3>Code:</h3>
<ul>
<li><a href="https://www.dropbox.com/sh/mqyj37mzk2coo52/AABnv9KtWLQ-dQYsnHhGdHroa?dl=0">code/import_arthropod_traits.r</a>, reads in trait file from Gossner et al. (‘data/arthropod_traits/ArthropodSpeciesTraits.csv’), interprets factor levels, save as matrix and plots correlations</li>
</ul>
</div>
<div id="notes-1" class="section level3">
<h3>Notes:</h3>
<p>Arthropod traits originate from the database compiled by Martin Gossner and Nadja Simons:</p>
<blockquote>
<p>Gossner MM, Simons NK, Achtziger R, Blick T, Dorow WHO, Dziock F, Köhler F, Rabitsch W, Weisser WW (2015) A summary of eight traits of Coleoptera, Hemiptera, Orthoptera and Araneae, occurring in grasslands in Germany. Scientific Data 2: 150013. <a href="http://dx.doi.org/10.1038/sdata.2015.13" class="uri">http://dx.doi.org/10.1038/sdata.2015.13</a> and Data <a href="http://dx.doi.org/10.5061/dryad.53ds2" class="uri">http://dx.doi.org/10.5061/dryad.53ds2</a></p>
</blockquote>
<p>With Araneae (177 species), Coleoptera (690), Hemiptera (331), and Orthoptera (32), this database is covering most of the species that occur in the abundance data and includes nine traits:</p>
<ul>
<li>body size [mm]</li>
<li>dispersal ability: defined as no dispersal (0), low passive dispersal (0.25), minor flying ability for movement and colonization (0.5), average individual movement and colonization dynamics (0.75) and high individual movment and colonization (floating, 1.0)</li>
<li>feeding guild: classified as consumer (c), detritivore (d), herbivore (h), fungivore (f), and omnivore (o)</li>
<li>feeding mode: classified as chewers (c), extraintestinal (e), and suction feeding (s)</li>
<li>feeding specialisation (within herbivores):</li>
<li>feeding tissue (within herbivorous suckers):</li>
<li>feeding plant part (within herbivorous chewers):</li>
<li>endophagous lifestyle (within herbivores):</li>
<li>stratum use: ground (g), stem (s),</li>
</ul>
<p>Only body size is a numeric trait (average adult body length from literature). All others are based on factorial classes. Feeding traits are only meaningful for herbivorous species.</p>
<p>Nadja is adding some missing traits to this database and will upload a revised file to bexis soon, using the new traitdata template. A preliminary version is available in the dropbox.</p>
<p>More trait data on butterflies, cicadas, and beetles have been provided by David Perovic and Catrin Westphal but those are only for exploration, since the original data owners have not been informed (e.g. Carmen Boerschig, Klaus Birkhofer, Martin Gossner). Please confirm with original data owners first and encourage upload to BExIS!</p>
</div>
</div>
<div id="plant-traits" class="section level2">
<h2>Plant traits</h2>
<p>Plant traits were extracted from the TRY database (Kattge et al. 2011). We requested trait measurements for all plant species that occurred in our vegetation assessments of 2008-2014 for six important plant traits that have been shown to span the major economic trade-offs of plant physiology (Diaz et al. 2016; specific leaf area, leaf C to N ratio, leaf dry mass, plant height, seed dry mass, stem specific density).</p>
<div id="issues-2" class="section level3">
<h3>Issues:</h3>
<ul>
<li>TRY request was based on plant list of 2008-2014, which had some major taxon name issues (refer to Debi and Dani) which were resolved in datasets of 2016. Maybe neccessary to re-do request with current species list!</li>
<li>ommit data that request co-authorship. Excluding them has a negligible effect on the resultant mean trait values.<br />
</li>
<li>Several data-quality issues in TRY data must be resolved, e.g. for non-numerical entries, for taxon name matching (e.g. agg.species), selection of outliers. I had a detailled conversation with Malte about that.<br />
</li>
<li>choice of traits? check selection and computation</li>
<li>How to handle missing trait data? could be inferred or just ignored. Which effect would that have on the CWM values?</li>
<li>eliminate duplicate entries via ObservationID</li>
</ul>
</div>
<div id="datasets-2" class="section level3">
<h3>Datasets:</h3>
<table style="width:64%;">
<colgroup>
<col width="16%" />
<col width="6%" />
<col width="15%" />
<col width="25%" />
</colgroup>
<thead>
<tr class="header">
<th>dataset</th>
<th>URL</th>
<th>BExIS ID</th>
<th>author/owner</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>TRY request 2181</td>
<td><a href="https://www.dropbox.com/sh/mqyj37mzk2coo52/AABnv9KtWLQ-dQYsnHhGdHroa?dl=0">/data/plant_traits_TRY/2181.txt</a></td>
<td></td>
<td>see <a href="https://www.dropbox.com/sh/mqyj37mzk2coo52/AABnv9KtWLQ-dQYsnHhGdHroa?dl=0">/data/plant_traits_TRY/dataset_reference.csv</a></td>
</tr>
</tbody>
</table>
<p>Data usage is subject to TRY Intellectual Property Guidelines and Data Release Notes (see Dropbox folder).</p>
</div>
<div id="code-2" class="section level3">
<h3>Code:</h3>
<ul>
<li><a href="https://www.dropbox.com/sh/mqyj37mzk2coo52/AABnv9KtWLQ-dQYsnHhGdHroa?dl=0">code/import_plant_traits.r</a>: reads 2181.txt; harmonizes factor levels; identifiers outliers and compares to Error Risk, eliminates outliers, eliminates bad datasets; averageiing traits per dataset, then per plant species; build and save plant-trait-matrix</li>
</ul>
</div>
<div id="notes-2" class="section level3">
<h3>Notes:</h3>
<p>Trait measurements for all plant species found in the years 2008-2013 were requested from TRY for the following traits:</p>
<p>1, 4, 55, 14, 26, 18, 8, 29, 6, 11, 679, 47, 46, 15</p>
<table>
<thead>
<tr class="header">
<th>TraitID</th>
<th>Trait</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1</td>
<td>Leaf area</td>
</tr>
<tr class="even">
<td>11</td>
<td>Leaf area per leaf dry mass (specific leaf area, SLA)</td>
</tr>
<tr class="odd">
<td>13</td>
<td>Leaf carbon (C) content per leaf dry mass</td>
</tr>
<tr class="even">
<td>55</td>
<td>Leaf dry mass</td>
</tr>
<tr class="odd">
<td>47</td>
<td>Leaf dry mass per leaf fresh mass (Leaf dry matter content, LDMC)</td>
</tr>
<tr class="even">
<td>14</td>
<td>Leaf nitrogen (N) content per leaf dry mass</td>
</tr>
<tr class="odd">
<td>15</td>
<td>Leaf phosphorus (P) content per leaf dry mass</td>
</tr>
<tr class="even">
<td>46</td>
<td>Leaf thickness</td>
</tr>
<tr class="odd">
<td>18</td>
<td>Plant height</td>
</tr>
<tr class="even">
<td>8</td>
<td>Plant nitrogen fixation capacity</td>
</tr>
<tr class="odd">
<td>679</td>
<td>Plant palatability</td>
</tr>
<tr class="even">
<td>29</td>
<td>Pollination syndrome</td>
</tr>
<tr class="odd">
<td>6</td>
<td>Rooting depth</td>
</tr>
<tr class="even">
<td>26</td>
<td>Seed dry mass</td>
</tr>
<tr class="odd">
<td>4</td>
<td>Stem dry mass per stem fresh volume (stem specific density, SSD)</td>
</tr>
</tbody>
</table>
<p>The species list of the core plant assessment was used to request data from TRY. I mapped the names of the datasets to the names accepted by TRY and requested taxon IDs:</p>
<ul>
<li>4045, 5134, 7173, 10820, 11785, 15171, 16700, 25526, 28295, 28473, 33153, 34017, 37495, 41549, 42541, 42893, 45587, 47451, 50209, 53415, 54175, 54643, 54991, 54998, 55161, 56236, 56242, 56396, 2172, 4036, 9957, 10304, 16337, 24558, 24604, 24735, 37504, 42546, 54945, 56277, 2389, 2872, 13878, 25247, 26081, 26097, 29305, 909, 35825, 55801, 56395, 32229, 1439, 7471, 26336, 45737, 55598, 9970, 17041, 42065, 9967, 32245, 45613, 10226, 26084, 47468, 47498, 15613, 32499, 34660, 45633, 54346, 42544, 51627, 1810, 8440, 25541, 25573, 31809, 34249, 34480, 37458, 39210, 42069, 44527, 46070, 46078, 48233, 49566, 56280, 17365, 26152, 45470, 51634, 3925, 5558, 8361, 10401, 10460, 10573, 10746, 10752, 13010, 23387, 24510, 25997, 28293, 29011, 29208, 31556, 31942, 32723, 32728, 33635, 43114, 43663, 43675, 43736, 48641, 49836, 54265, 54936, 54974, 7316, 8151, 12062, 31934, 45061, 53962, 4072, 30010, 61831, 42185, 44078, 48156, 13016, 23666, 38993, 11930, 54165, 56252, 26083, 50304, 10578, 10682, 13051, 14244, 9975, 30052, 13072, 4614, 13032, 26052, 47518, 973, 39153, 41886, 42575, 1905, 39302, 11771, 15090, 28112, 46728, 4498, 49532, 43716, 44304, 56402, 4491, 21917, 25385, 55130, 56356, 40533, 2065, 37517, 43221, 11583, 12917, 44419, 45601, 50169, 35846, 38997, 56526, 20359, 34016, 11635, 20927, 41971, 56372, 8542, 32511, 56268, 2548, 41893, 46948, 8455, 7297, 23910, 4489, 13049, 18729, 54919, 44422, 2165, 5849, 6903, 25213, 3959, 25923, 26069, 47452, 43637, 10497, 41183, 41425, 52530, 25487, 2854, 31454, 31479, 10704, 24150, 10312, 10355, 20877, 50913, 33555, 56376, 21900, 23920, 41658, 43638, 2590, 20891, 26473, 23624, 55773, 53949, 3738, 32132, 26007, 14280, 50220, 42574, 54965, 15115, 32771, 42805, 56581, 50347, 56491, 21961, 4185, 8867, 32490, 48601, 41193, 21625, 12159, 32240, 21995, 24995, 54926, 4497, 3455, 23444, 11800, 15200, 50873, 9139, 36281, 868, 56247, 7356, 10626, 11938, 25215, 10598, 48631, 31490, 10773, 4357, 9959, 47353, 11796, 8266, 35678, 35672, 3462, 25355, 9949, 10594, 26145, 17587, 11783, 51535, 56262, 43715, 37499, 50348, 8262, 12236, 58995, 50427, 4341, 3691, 7875, 56398, 39048, 33718, 49451, 50934, 15080, 49407, 49914, 56508, 20845, 29984, 38977, 84098, 36265, 56588, 11617, 53018, 30000, 34666, 29545, 43532, 32357, 38688,</li>
</ul>
<p>The received data were filtered for outliers, regional match, and aggregated into species average values. The TRY database holds replicated measurements of trait data from multiple authors, partly following different protocolls of measurement. The high replication is what is powerful in TRY, since it allows us to estimate a general average for a plant species.</p>
<p>To reduce the list of datasets for your analysis based on the conditions of the data owners, I reviewed the importance of individual datasets for your project and decide which data to involve. Fortunately, for grasslands only about 6% of the raw data request co-authorship for our traits and species. Most trait values are not significantly altered if those datasets are excluded. When analysing the trait values produced with and without these data, only leaf thickness significantly relies on it (contributing 36 of 93 species and adding 12% accuracy to the observed values of the remaining ones). In studies focusing on this trait, the following datasets might be included and the authors offered coauthorship: “Global Leaf Robustness and Physiology Database”, “Plant Traits for Grassland Species (Konza Prairie, Kansas, USA)” and “Roots Of the World (ROW) Database”. The data for leaf Nitrogen content receive 5 of 237 species 2% of accuracy from data with coauthorship request (mostly by the “Catalonian Mediterranean Forest Trait Database” and the “Global 15N Database”) which may be included for accuracy. The data for rooting depth contain 2 more species, while affecting 2 further species significantly.</p>
<p>To my understanding, it is not necessary to write a new proposal after filtering the dataset. But to comply with the TRY terms, all authors must be informed according to their preset conditions. In any case, for trannsparency and reproducibility of the study, we will have to publish a list of datasets included in your analysis, e.g. as supplementary information. This list can be extracted from the TRY <a href="https://www.try-db.org/TryWeb/Prop01.php">Request PI Center</a>, as well.</p>
<p>The TRY data contain a column <code>ErrorRisk</code> which reports the number of standard deviations that the measured value is away from the global mean. Since this is a global metric, provided by some computation on the TRY database server, it might not reflect exactly the criterion of an outlier. Instead, the code ‘import_plant_traits.r’ manually remove outliers from the data before averaging, i.e. measurements that are outside the 95% quantile of all replicated measurements (paper by Craven et al ?).</p>
<p>We remove datasets of authors that produced a significant amount of those outliers. There is a chance that the authors used a non-standard method or did not standardise their units according to TRY. To identify these datasets, we extract the proportion of outliers in the total measurement contribution of this dataset.</p>
<p>Large trees and shrubs should be excluded from the assessment of community weighted means in grasslands. Their trait values reflect adult plants of several meters height. The individuals found in the plant survey on the grassland plots are seedlings or saplings that are not well represented by these values. A common practice is to exclude all species that have trait values of plant height &gt; 2m.</p>
<p>The following species should be particularly excluded:</p>
<pre><code>&quot;Betula pendula&quot;, &quot;Carpinus betulus&quot;, &quot;Crataegus laevigata&quot;, &quot;Crataegus monogyna&quot;, 
&quot;Crataegus sp&quot;, &quot;Fraxinus excelsior&quot;, &quot;Juniperus communis&quot;, &quot;Pinus sylvestris&quot;, 
&quot;Populus tremula&quot;, &quot;Prunus avium&quot;, &quot;Prunus sp&quot;, &quot;Prunus spinosa&quot;, &quot;Quercus robur&quot;,
&quot;Rosa canina&quot;, &quot;Acer sp&quot;, &quot;Baumkeimling sp&quot;, &quot;Tilia sp&quot;</code></pre>
<p>An exception here is Juniperus, which as Dani pointed out may be considered an important contribution to the functional diversity in xeric grassland plots. Depending on the research question and spatial scale, Juniperus should be included.</p>
<p>Since each project or author likely measured specimens that were co-varying because of regional or environmental differences, we recommend to not over-rating the heavily repeated measures of one author against the few measures of another author. Therefore, we average the values measured for each plant per author and use these averages to form an overall average for each plant species. Those averages then can be compiled into a matrix reporting one value per trait per species.</p>
<p>A potential extension of these data could be root traits: FRED database, 2017. In addition to the abundance data, the group of Helge Bruelheide investigates plant root traits and their interaction with community parameters. They might be able to provide data and detailled knowledge. If we could add the root traits as a complement to the aboveground plant traits, this might add another strong explanatory element to the correlation between plants and below-ground organisms.</p>
</div>
</div>
<div id="functional-soil-properties" class="section level2">
<h2>Functional soil properties</h2>
<p>For soil functional properties, data were gathered in concerted soil sampling campaigns in the years 2011 and 2014 for all 150 plots, as well as 2008 for some selected plots in each of the three regions. Soil samples were cored out (core size 5 cm), the top most 5 cm root-layer removed as well as any deadwood and roots larger than 2 cm in diameter. Samples were homogenized and stored at 4°C while still at field moisture content.</p>
<p>The community-level properties were measured from pooled soil samples, yielding micobial carbon to nitrogen ratio, relative abundance of bacteria and fungi, as well as invertebrates based on phospholipid-fatty-acids (PLFA), and different enzyme activities (Urease, Phosphatase, beta-Glucosidase, Xylosidase and Chitinase).</p>
<p>In addition to these metrics, the soil sampling in 2011 and 2014 produced relative abundances for major taxa of bacteria and fungi. Metagenomes were extracted using PCR-mediated amplification (using rDNA fungal primers ITS1F and ITS4, and 16S rRNA bacterial primers V3-V5) and pyrosequencing. Sequences were mapped into relative abundances for operational taxonomic units (OTUs) from databases for fungi (UNITE, version 6) and bacteria (SILVA SSURef NR 123 Database), which can be linked to certain functional processes in soils.</p>
<div id="issues-3" class="section level3">
<h3>Issues:</h3>
<ul>
<li>include 2008 VIP data for Cmic/Nmic ratio? (missing for Schorfheide on BExIS)</li>
<li>get fungal data for 2014 (Tesfaye?)</li>
<li>verify date of sampling for Kaiser/Daniel data. I think they may be mistaken!</li>
<li>get OTU relative abundance for soil-sampling 2014 (Kristin Kaiser, Kezia Goldman, Tesfaye Wubet)</li>
<li>define community-level function based on OTU for bacteria (Fierer et al. 2006) and fungi (based on what information?)</li>
</ul>
</div>
<div id="datasets-3" class="section level3">
<h3>Datasets:</h3>
<table style="width:56%;">
<colgroup>
<col width="12%" />
<col width="9%" />
<col width="13%" />
<col width="19%" />
</colgroup>
<thead>
<tr class="header">
<th>dataset</th>
<th>DOI</th>
<th>BExIS ID</th>
<th>author/owner</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>SSC 2011, EP grassland, soil enzyme activities, SCALEMIC</td>
<td></td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=20246">20246</a></td>
<td>Runa Boeddinghaus, Ellen Kandeler, Sven Marhan, et al.</td>
</tr>
<tr class="even">
<td>SSC 2014, EP grassland, soil enzyme activities, SCALEMIC</td>
<td></td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=20247">20247</a></td>
<td>Runa Boeddinghaus, Ellen Kandeler, Sven Marhan, et al.</td>
</tr>
<tr class="odd">
<td>SSC 2011, EP grassland, microbial soil properties, SCALEMIC</td>
<td></td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=20250">20250</a></td>
<td>Runa Boeddinghaus, Ellen Kandeler, Sven Marhan, et al.</td>
</tr>
<tr class="even">
<td>SSC 2014, EP grassland, microbial soil properties, SCALEMIC</td>
<td></td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=20251">20251</a></td>
<td>Runa Boeddinghaus, Ellen Kandeler, Sven Marhan, et al.</td>
</tr>
<tr class="odd">
<td>Bacterial functional profile (predicted metagenome)</td>
<td></td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=20346">20346</a></td>
<td>Kristin Kaiser, Rolf Daniel</td>
</tr>
<tr class="even">
<td>Bacterial Abundances based on partial (V3-V5) 16S rRNA genes</td>
<td></td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=19526">19526</a></td>
<td>Kristin Kaiser, Rolf Daniel</td>
</tr>
<tr class="odd">
<td>EP-Grassland-AMF-SoilCampain2011</td>
<td></td>
<td>19786</td>
<td>Tesfaye Wubet, Kezia Goldmann</td>
</tr>
<tr class="even">
<td>EP-Grassland abundant Soil fungi from Soil Sampling Campain 2011</td>
<td></td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=21048">21048</a></td>
<td>Tesfaye Wubet, Kezia Goldmann</td>
</tr>
</tbody>
</table>
</div>
<div id="code-3" class="section level3">
<h3>Code:</h3>
<ul>
<li><a href="https://www.dropbox.com/sh/mqyj37mzk2coo52/AABnv9KtWLQ-dQYsnHhGdHroa?dl=0">code/data_soil.R</a>: reads microbial soil properties (20250, 20251), calculate gram neg/pos ratio, save file; reads fungal profile (21048), produces plot matrix.</li>
</ul>
</div>
<div id="notes-3" class="section level3">
<h3>Notes:</h3>
<p>Runa Boeddinghaus and Ellen Kandeler are main responsible authors for the microbial and PLFA data. In the two soil sampling campaigns in 2011 and 2014, they produced</p>
<ul>
<li>soil moisture</li>
<li>bulk density</li>
<li>Mineral nitrogen (nitrate and ammonium)</li>
<li>Phospholipid fatty acid analysis (fungi, gram positive and gram negative bacteria, total bacteria); yields bacterial/fungal biomass ratio, Invertebrate markers</li>
<li>Microbial biomass carbon and nitrogen (Cmic and Nmic from CFE analysis); microbial C/ microbial N ratio, values ~5 indicate Bacterial dominance, which indicates fast turnover soils which usually is associated with nutrient and fertilizer addition</li>
<li>Ergosterol (fungal membrane marker)</li>
<li>Extractable organic carbon and nitrogen (EOC and EON)</li>
<li>Enzyme activities:
<ul>
<li>Urease: responds to fertilization and grazing activity</li>
<li>Phosphatase: Acidity, responds to pH</li>
<li>beta-Glucosiddase: litter decomposition and cellulose decomposition</li>
<li>beta-Xylosidase: hemicellulose decomposition, litter</li>
<li>Chitinase: Carbon decomposition, litter from arthropods</li>
<li>denitrification enzyme activity (DEA)</li>
</ul></li>
</ul>
<p>In 2008 enzyme activities, microbial biomass, EON, EOC, and mineral nitrogen were measured only on the VIP grassland plots. PLFA analysis haven’t been done back then.</p>
<p>Runa was fixing some errors with the datasets, so please make sure you download the most recent versions!</p>
<p>The metagenomic analysis for the soil sampling campaigns based on 16S rRNA has been done by Kristin Kaiser (PhD student) and Rolf Daniel (PI, Göttingen). The data on BExIS are providing abundances of taxa based on a mapping of partial (V3-V5 region) 16S rRNA gene amplicons to the SILVA SSURef NR 123 Database. They further offer functional profiles based on KEGG orthology (Kyoto Encyclopedia of Genes and Genomes), i.e. gene sequences specific for certain enzymes or processes. The predictive metagenome was calculated for the bacterial taxonomy. The sampling was carried out during the joint sampling campaign in May 2012, i.e. not in the same date as the other soil data?!</p>
<p>Starting 2011, fungal analysis has been done by the group of Tesfaye Wubet, with Kezia Goldmann as the PhD Student. They provide abundances for all fungal taxa and for arbuscular mycorhyzal fungi in particular (both based on OTUs) found on the 150 grassland EPs for 2011. 2014 data are in the making. The dataset on BExIS for 2011 contains functional properties of the taxa, which can be extracted into a binary trait-matrix and used to compute CWMs for each plot.</p>
</div>
</div>
<div id="below-ground-consumers" class="section level2">
<h2>Below ground consumers</h2>
<p>We further are looking into relative abundance data and functional properties of protist communities. The diversity of two major groups of amoebal protists (Myxomycetes, Acanthamoeba) was investigated using 454 sequencing in all grasslands plots in the three exploratories. The first part of the SSU was targeted using specifically designed primers.</p>
<div id="issues-4" class="section level3">
<h3>Issues:</h3>
<ul>
<li>get OTU relative abundances for 2014? for both amoeba and ciliates/flaggelates.</li>
<li>get functional interpretation of Amoeba data (Michael Bonkowski)</li>
<li>get traits of other protists (Hartmut Arndt)</li>
</ul>
</div>
<div id="datasets-4" class="section level3">
<h3>Datasets:</h3>
<table style="width:62%;">
<colgroup>
<col width="12%" />
<col width="15%" />
<col width="34%" />
</colgroup>
<thead>
<tr class="header">
<th>dataset</th>
<th>BExIS ID</th>
<th>authors/owner</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Acanthamoeba NGS abundance (AEG, 2011 - 2012)</td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=18166">18166</a></td>
<td>Anna Maria Fiore-Donno, Michael Bonkowski</td>
</tr>
<tr class="even">
<td>Acanthamoeba NGS abundance (HEG, 2010 - 2011)</td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=18207">18207</a></td>
<td>Anna Maria Fiore-Donno, Michael Bonkowski</td>
</tr>
<tr class="odd">
<td>Acanthamoeba NGS abundance (SEG, 2011 - 2012)</td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=18208">18208</a></td>
<td>Anna Maria Fiore-Donno, Michael Bonkowski</td>
</tr>
<tr class="even">
<td>Myxomycetes NGS abundance (AEG, 2011 - 2012)</td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=18226">18226</a></td>
<td>Anna Maria Fiore-Donno, Michael Bonkowski</td>
</tr>
<tr class="odd">
<td>Myxomycetes NGS abundance (HEG, 2011 - 2012)</td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=18187">18187</a></td>
<td>Anna Maria Fiore-Donno, Michael Bonkowski</td>
</tr>
<tr class="even">
<td>Myxomycetes NGS abundance (SEG, 2011 - 2012)</td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=18206">18206</a></td>
<td>Anna Maria Fiore-Donno, Michael Bonkowski</td>
</tr>
<tr class="odd">
<td>OTUs on the basis of 99.7% direct sequence similarity with PR2Database reference sequences</td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=20326">20326</a></td>
<td>Paul Venter, Hartmut Arndt</td>
</tr>
</tbody>
</table>
</div>
<div id="code-4" class="section level3">
<h3>Code:</h3>
<p>no import of these data has taken place, yet.</p>
</div>
<div id="notes-4" class="section level3">
<h3>Notes:</h3>
<p>Michael Bonkowski gave this reply: “First of all, our sequencing nearly reached saturation - that means the sequences per site are not found due to merely chance events, but reflect the ‘real’ diversity in that group. Second, both groups are phylogenetically highly diverse, and they showed high beta diversity between grassland sites. If you need it, we can also assign real taxonomic names to the OTUs, although as often for protists, some”species&quot; are species clusters with broad phylogenetic dispersion, and other “species” occur repeatedly on distant branches in the phylogeny, showing that they are in fact several different taxa. Therefore “OTU” is more reliable for non-experts.&quot;</p>
<p>The data are fully accessible as supplementary material in the article:</p>
<blockquote>
<p>Fiore-Donno AM, Weinert J, Wubet T &amp; Bonkowski M. 2016. Metacommunity analysis of amoeboid protists in grassland soils. Sci Rep, 6:19068. doi: <a href="http://www.nature.com/articles/srep19068">10.1038/srep19068</a>.</p>
</blockquote>
<p>For other protists of the RAS-group (Rhizaria, Alveolata, Stramenopiles), Hartmut Arndt and Paul Venter have OTU data approximately resolved to species. They are willing to provide trait data including biovolume (related to reproductive potential), length, and function (differentiating autotroph and heterotroph, and within the latter the kind of resource e.g. feeding on algae, grazing on bacteria, suspension feeders, feeding on protists), as well as relative abundances. However, the data are not yet compiled.</p>
</div>
</div>
<div id="anthropogenic-land-use" class="section level2">
<h2>Anthropogenic Land-use</h2>
<p>As explanatory variables, our study aims at the role of the intensity of land use on functional composition. The Land-use Intensity Index (LUI) defines the summed relative intensity of multiple land use factors (i.e. livestock grazing pressure, mowing events and amount of fertilizer added) applied on a plot of land in the year of the assessment.</p>
<div id="issues-5" class="section level3">
<h3>Issues:</h3>
<ul>
<li>using standardised (within Regions) LUI vs non-standardised LUI?</li>
<li>using seperate Land use factos (grazing, mowing, fertilization)?</li>
</ul>
</div>
<div id="datasets-5" class="section level3">
<h3>Datasets:</h3>
<table>
<thead>
<tr class="header">
<th>dataset</th>
<th>BExIS ID</th>
<th>authors/owner</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Input LUI Calculation Tool</td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=19266">19266</a></td>
<td>Katrin Lorenzen, Wolfgang Weisser et al.</td>
</tr>
</tbody>
</table>
</div>
<div id="notes-5" class="section level3">
<h3>Notes:</h3>
<p>The LUI is available via the <a href="https://www.bexis.uni-jena.de/LuiTool/LuiTool.aspx?DatasetId=19266">LUI download tool</a> or as raw dataset download (<a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=19266">19266</a>). The tool lets you select standardised or not standardised by region.</p>
</div>
</div>
<div id="environmental-parameters" class="section level2">
<h2>Environmental parameters</h2>
<p>Being independent of anthropogenic pressure, we also assess the effect of climate and environmental factors on the composition of traits in grassland communities. We assembled topographic parameters of the plots (elevation, slope, topographic wetness index, TWI) and soil physical parameters (Soil pH, …) as explanatory variables.</p>
<div id="issues-6" class="section level3">
<h3>Issues:</h3>
<ul>
<li>get official data (BExIS source)</li>
<li>standardising variables for region to cover climatic effects (or not)?</li>
<li>topo data are constant through time, i.e. don’t explain any annual changes! in addition get annual climate data (precipitation?, temperature?, pH?).</li>
</ul>
</div>
<div id="datasets-6" class="section level3">
<h3>Datasets:</h3>
<table style="width:43%;">
<colgroup>
<col width="11%" />
<col width="11%" />
<col width="20%" />
</colgroup>
<thead>
<tr class="header">
<th>dataset</th>
<th>BExIS ID</th>
<th>author/owner</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Basic Information of all Experimental Plots (EPs)</td>
<td><a href="https://www.bexis.uni-jena.de/Data/ShowXml.aspx?DatasetId=20826">20826</a></td>
<td>Andreas Ostrowski, Birgitta König-Ries</td>
</tr>
<tr class="even">
<td>GP Soils and Carbon stocks master-sheet_with terrain</td>
<td><a href="https://www.dropbox.com/sh/mqyj37mzk2coo52/AABnv9KtWLQ-dQYsnHhGdHroa?dl=0">data/environment/GP Soils and Carbon stocks master-sheet_with terrain.xlsx</a></td>
<td>Pete Manning (unknown primary source)</td>
</tr>
</tbody>
</table>
</div>
<div id="code-5" class="section level3">
<h3>Code:</h3>
<ul>
<li><a href="https://www.dropbox.com/sh/mqyj37mzk2coo52/AABnv9KtWLQ-dQYsnHhGdHroa?dl=0">import_env_data.r</a> calls topo_data.txt and soil_data.txt, which have been manually created from Petes file.</li>
</ul>
</div>
<div id="notes-6" class="section level3">
<h3>Notes:</h3>
<p>No direct import of these data has taken place yet. I was using data on soil topology that I received from Pete (data/environment/GP Soils and Carbon stocks master-sheet_with terrain.xlsx). which include plot level elevation, slope, aspect, soil profile, litter layer C and N, soil type, topographic wetnes index (TWI) as calculated from a DEM, and more.</p>
<p>I assume this is a merged dataset from Bexis datasets: - 6223 : Soil: CN gridplots - 6226 : Soil: pH-analysis grid plots - 10022 : Digital Terrain Models of all Exploratories</p>
<p>The data contain all plots, thus need to be filtered for EPs and matched EP IDs.</p>
<p>I manually extracted topographical parameters (TWI, slope, elevation, aspect) for the EP plots.</p>
</div>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
